core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -194,6 +195,8 @@ private interface ClientRequestMetrics extends RequestMetrics {
         Counter writeTimeouts();
 
         Counter responseTimeouts();
+
+        DistributionSummary retries();
```

Comment:
Note: although the name is `retries` this will also be incremented when redirect requests are made.
Having said this, I can't think of a better name for this 😅

Reply:
Maybe `attempts`?